add_to_app/flutter_module/pubspec.lock

```diff
@@ -271,13 +271,6 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "2.1.2"
-  xml:
-    dependency: transitive
-    description:
-      name: xml
-      url: "https://pub.dartlang.org"
-    source: hosted
-    version: "3.6.1"
 sdks:
-  dart: ">=2.7.0 <3.0.0"
+  dart: ">=2.9.0-14.0.dev <3.0.0"
```

Comment:
What SDK version were you using when you did `flutter pub upgrade`? Are you trying to sneak the master channel back into my beautiful, stable sample app? 😄

Reply:
ah doh, that's what I was doing. 

This conversation never happened